src/test/regress/input/multi_copy.source

```diff
@@ -254,3 +254,16 @@ SET citus.shard_max_size TO '256kB';
 COPY lineitem_copy_append FROM '@abs_srcdir@/data/lineitem.1.data' with delimiter '|';
 
 SELECT count(*) FROM pg_dist_shard WHERE logicalrelid = 'lineitem_copy_append'::regclass;
+
+-- Test worker copy on the master node.
+-- Note that worker copy creates a temporary relation on the local node. We already
+-- have same table on the master node which is also local node for the regression tests.
+-- This should not be a problem because we create a temporary table which has a
+-- special schema.
+COPY lineitem_copy_append FROM '@abs_srcdir@/data/lineitem.1.data' with (delimiter '|', master_host 'localhost', master_port 57636);
```

Comment:
Should we also do these tests from a worker?

You can connect to a worker by running:

    \c - - - :worker_1_port